bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutatorHelper.java

```diff
@@ -204,10 +204,10 @@ public long getWriteBufferSize() {
         future = bulkMutation.add(adapter.adaptEntry((Delete) mutation));
       } else if (mutation instanceof Increment) {
         future = dataClient.readModifyWriteRowAsync(adapter.adapt((Increment) mutation));
-        operationAccountant.registerOperation(ApiFutureUtil.adapt(future));
+        operationAccountant.registerOperation(new ApiFutureToListenableFuture<>(future));
```

Comment:
Can we avoid the wrapping here altogether? and just have OperationAccountant work with ApiFutures?

Reply:
marking as resolved as address this on #2468